FrontMen/fm-hours
# feat(insights-for-employees): component added and translations

# Changes

#560 

## Added

- Reusable component "date-reports.vue"
- Helper function to getMonthName
- Page for yearly reports
- Button in the header that will lead to the yearly report (where you can also see the page yearly title)

## Changed

- You can now see at the page title the month which you are viewing.
- The /month route is now using the new "date-reports.vue" cmp.

## How to test

1. Login
2. At the header you will see a button called "Yearly overview", that should show you your yearly report, you can go to any year you want. 

## Screenshots

![image](https://user-images.githubusercontent.com/46437510/171123291-8595675f-dd8e-4e43-b5e6-02c31b6ecded.png)
